src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -98,70 +107,101 @@ limitations under the License.
       <feature bit="0" code="EXTS" name="Extension" summary="Device provides ACL Extension attribute">
         <optionalConform/>
       </feature>
-      <feature bit="1" code="MNGD" name="Managed Device" summary="Device is managed">
-        <optionalConform/>
-      </feature>
+      <feature bit="1" code="MNGD" name="ManagedDevice" summary="Device is managed"/>
```

Comment:
Why is this removing the "optionalConform"?  Spec has Conformance as "desc", which is very much not the same thing as "required".

Reply:
What does `desc` mean in conformance? I did not see it's explained in the core spec. This irregularity might make Alchemy not parsing it correctly

Reply:
"desc" in conformance means "the actual conformance is described as English prose, sucks to be you if you need machine-readable conformance".

But typically it does NOT mean "M", because if it were "M" it would just say "M".  It typically means some sort of "required not always but under some sort of complicated circumstances".

Reply:
Note that this applies to the event priority too: whenever the spec says "desc" that means "not machine readable, sorry".

Reply:
Thank you for the explanation!